pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -2424,6 +2456,18 @@ func (a *ApplicationBackupController) IsVolsToBeBackedUp(backup *stork_api.Appli
 	return true
 }
 
+// IsPVCInExcludeResourceType check if given ResourceType is PVC
+func IsPVCInExcludeResourceType(backup *stork_api.ApplicationBackup) bool {
+	for _, resType := range backup.Spec.ExcludeResourceTypes {
+		if resType == "PersistentVolumeClaim" {
+			return true
+		} else {
```

Comment:
This else is. redundant check can be removed completely

Reply:
Fixed